NuRadioReco/modules/io/coreas/coreas.py

```diff
@@ -426,63 +392,62 @@ def create_sim_station(station_id, evt, weight=None):
     return sim_station
 
 
-def make_sim_shower(corsika, observer=None, detector=None, station_id=None):
+def make_sim_shower(corsika, declination=0):
```

Comment:
I think this function should have another name to allow for a deprecation warning. As mentioned before, I think it should be part of read_Corsika.